### PR TITLE
Mocks and endpoints connectivity section 3 geometries

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: services/backend/src/routes/connectivity.js [hunk 1]
```diff
@@ -80,7 +80,8 @@ module.exports = (errorHandler, connectivityService) => {
    * @apiName DPCLayer
    * @apiVersion 1.0.0
    * @apiDescription
-   * Layers of the protected areas with higher dPC value in a given area
+   * Layers of the protected areas with higher dPC value in a given area. If paNumber is not
```
Comment (erikasv) [inline services/backend/src/routes/connectivity.js:83]: Mark the param as optional in the documentation

File: services/backend/src/routes/connectivity.js [hunk 2]
```diff
@@ -192,5 +193,44 @@ module.exports = (errorHandler, connectivityService) => {
       });
   }));
 
+  /**
+   * @apiGroup s_pa_connectivity
+   * @api {get} /connectivity/se/layer SELayer
+   * @apiName SELayer
+   * @apiVersion 1.0.0
+   * @apiDescription
+   * Layer of a estrategic ecosystem in a given area
```
Comment (erikasv) [inline services/backend/src/routes/connectivity.js:202]: strategic

File: services/backend/src/service/connectivity.js [hunk 3]
```diff
@@ -113,6 +123,41 @@ module.exports = () => {
 
       return data;
     },
+
+    /**
+     * Get the layer of a estrategic ecosystem in a given area
```
Comment (erikasv) [inline services/backend/src/service/connectivity.js:128]: strategic

**Review by erikasv — CHANGES_REQUESTED** [review]

File: services/backend/src/routes/connectivity.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ module.exports = (errorHandler, connectivityService) => {
    * @apiDescription
    * Area distribution for each category of protected area connectivity in a given area
    *
-   * Value calculated for 2018
+   * Value calculated for 2020
    *
    * @apiParam (Query params) {String|Number} areaType area type
```
Comment (Trjegul84) [inline services/backend/src/routes/connectivity.js:16]: According previous reviews areaType is String and the one that could be both String or Number is areaId

File: services/backend/src/routes/connectivity.js [hunk 5]
```diff
@@ -161,7 +162,7 @@ module.exports = (errorHandler, connectivityService) => {
    * Area distribution for each category of protected area connectivity for an specific strategic
    * ecosystem in a given area
    *
-   * Value calculated for 2018
+   * Value calculated for 2020
    *
    * @apiParam (Query params) {String|Number} areaType area type
```
Comment (Trjegul84) [inline services/backend/src/routes/connectivity.js:167]: areaType -> String
areaId  -> String or Number

File: services/backend/src/routes/connectivity.js [hunk 6]
```diff
@@ -192,5 +193,44 @@ module.exports = (errorHandler, connectivityService) => {
...
+   * @apiGroup s_pa_connectivity
+   * @api {get} /connectivity/se/layer SELayer
+   * @apiName SELayer
+   * @apiVersion 1.0.0
+   * @apiDescription
+   * Layer of a estrategic ecosystem in a given area
+   *
+   * Value calculated for 2020
+   *
+   * @apiParam (Query params) {String|Number} areaType area type
```
Comment (Trjegul84) [inline services/backend/src/routes/connectivity.js:206]: Check again the param types of areaType and areaId

File: services/backend/src/service/connectivity.js [hunk 7]
```diff
@@ -113,6 +123,41 @@ module.exports = () => {
...
+     *
+     * @param {String | Number} areaType area type
```
Comment (Trjegul84) [inline services/backend/src/service/connectivity.js:130]: Check again the param types of areaType and areaId

File: services/backend/src/service/connectivity.js [hunk 8]
```diff
@@ -34,15 +38,21 @@ module.exports = () => {
     getPADPC: async () => connectivityDPC,
 
     /**
-     * Get the layers of the protected areas with higher dPC value in a given area
+     * Get the layers of the protected areas with higher dPC value in a given area. If paNumber
+     * is not provided, all layers are returned
      *
      * @param {String | Number} areaType area type
```
Comment (Trjegul84) [inline services/backend/src/service/connectivity.js:44]: Check again the param types of areaType and areaId

File: services/backend/src/routes/connectivity.js (hunk 1 above)
Comment (cazapatamar) [reply services/backend/src/routes/connectivity.js:83]: Done

File: services/backend/src/routes/connectivity.js (hunk 2 above)
Comment (cazapatamar) [reply services/backend/src/routes/connectivity.js]: Done

File: services/backend/src/service/connectivity.js (hunk 3 above)
Comment (cazapatamar) [reply services/backend/src/service/connectivity.js]: Done

File: services/backend/src/routes/connectivity.js (hunk 4 above)
Comment (cazapatamar) [reply services/backend/src/routes/connectivity.js]: Done

File: services/backend/src/routes/connectivity.js (hunk 5 above)
Comment (cazapatamar) [reply services/backend/src/routes/connectivity.js]: Done

File: services/backend/src/routes/connectivity.js (hunk 6 above)
Comment (cazapatamar) [reply services/backend/src/routes/connectivity.js]: Done

File: services/backend/src/service/connectivity.js (hunk 7 above)
Comment (cazapatamar) [reply services/backend/src/service/connectivity.js]: Done

File: services/backend/src/service/connectivity.js (hunk 8 above)
Comment (cazapatamar) [reply services/backend/src/service/connectivity.js]: Done

**Review by erikasv — APPROVED** [review]

**Review by Trjegul84 — APPROVED** [review]

**Pull request merged**